flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java

```diff
@@ -23,9 +23,9 @@
 import org.apache.flink.configuration.Configuration;
 import org.apache.flink.configuration.ConfigurationUtils;
 import org.apache.flink.configuration.HighAvailabilityOptions;
```

Comment:
Unused import that checkstyle complains with

Reply:
Good catch. Will fix it.